src/prefect/flows.py

```diff
@@ -203,6 +206,7 @@ def __init__(
         ] = None,
         on_crashed: Optional[List[Callable[[FlowSchema, FlowRun, State], None]]] = None,
         on_running: Optional[List[Callable[[FlowSchema, FlowRun, State], None]]] = None,
+        transaction_store: Optional[RecordStore] = None,
```

Comment:
This `transaction_store` kwargs is added to customize the record store implementation that should be used for child tasks.